src/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjema.tsx

```diff
@@ -6,32 +6,34 @@ import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
 import { VurderAktivitetskravSkjemaButtons } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjemaButtons";
 import { Form } from "react-final-form";
 import { useVurderAktivitetskrav } from "@/data/aktivitetskrav/useVurderAktivitetskrav";
-import { VurderAktivitetskravBeskrivelse } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravBeskrivelse";
 import { ValidationErrors } from "final-form";
 
-interface VurderAktivitetskravSkjemaProps<SkjemaValues> {
-  title: string;
-  subtitle?: ReactElement;
-  beskrivelseLabel?: string;
-  arsakVelger: ReactElement;
+export interface VurderAktivitetskravSkjemaProps {
```

Comment:
Skilt ut disse fra `Props` under siden disse er felles props for alle skjemaene som bruker `VurderAktivitetskravSkjema`.